app/models/accommodation.rb

```diff
@@ -47,7 +47,7 @@ def self.filter_holds(admin)
 
   def get_label
     if (self.building.building_type.label.eql?('Cabin') || self.building.building_type.label.eql?('Lodge'))
-      return "#{self.building.label} #{self.label}"
+      return "#{self.label}"
```

Comment:
I've confirmed that both when I book a cabin and when I book main lodge, I get the double "Cabin Cabin" or "Main Lodge Main Lodge" in the email.  Since the email simply calls this function for the label, I'm pretty sure this is the right fix?